baseframe/templates/baseframe/forms.html

```diff
@@ -247,4 +248,26 @@
     });
   </script>
   {%- endif %}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro add_map_marker_widget(field) %}
+  <style>
```

Comment:
Style shouldn't be inline. `height: 20em` isn't a safe assumption. This will need styling per responsive breakpoint.